### PR TITLE
Import Module

### PR DESCRIPTION
**Pull request by ingalls**
## Context

- [x] Fix bug where validation stream halt due to not being read by writing stream to disk
- [x] This unblocks havhing `options.input` be a stream, like all of the `options.output` values already are for far more consistent behavior (and much better behavior when used as a library)
- [x] Adds server capabilities check for max upload size
- [x] Add basic test for `lib/import` module

### CODE REVIEW

File: lib/import.js [hunk 1]
```diff
@@ -91,7 +58,7 @@ class Import {
      *
      * @param {Object} options options object
      * @param {string} options.message Human readable description of changes
-     * @param {string|Object[]} options.input String of filepath or Array containing features to import
+     * @param {Stream} options.input Stream of line-delimited geojson features to import
```
Comment (miccolis) [inline lib/import.js]: Looks like you accept a string that is the path to a file too right?

File: lib/import.js [hunk 2]
```diff
@@ -149,6 +122,12 @@ class Import {
                 options.ignoreDup = !!options['ignore-dup'];
                 options.dryrun = !!options.dryrun;
 
+                if (argv.input) {
```
Comment (miccolis) [inline lib/import.js]: It's strange to check `argv.input` and then use `options.input` - any reason for that?

File: lib/import.js [hunk 3]
```diff
@@ -182,24 +166,27 @@ class Import {
...
                     pipeline(
-                        fs.createReadStream(path.resolve(options.input)),
+                        options.input,
                         split(),
                         validateGeojson({
                             ignoreRHR: options.ignoreRHR,
                             schema: schema,
                             ids: !options.ignoreDup
                         }),
+                        fs.createWriteStream(tmp),
```
Comment (miccolis) [inline lib/import.js]: I think this deserves a comment.

File: lib/import.js (hunk 2 above)
Comment (ingalls) [reply lib/import.js]: Yes, argv is the response from the promp library, options is what is actually used to make the request

Comment (ingalls) [reply lib/import.js]: Good catch

File: lib/import.js (hunk 1 above)
Comment (ingalls) [reply lib/import.js]: I'll add support for a string for the API

**Pull request merged**